src/EditorFeatures/Core.Wpf/InlineRename/AbstractEditorInlineRenameService.cs

```diff
@@ -176,9 +159,12 @@ private IInlineRenameInfo GetRenameInfo(Document document, int position, Cancell
             var symbolLocations = symbol.Locations;
 
             // Does our symbol exist in an unchangeable location?
-            var navigationService = workspace.Services.GetService<IDocumentNavigationService>();
+            var documentSpans = new List<DocumentSpan>();
             foreach (var location in symbolLocations)
             {
+                // We eventually need to return the symbol locations, so we must convert each location to a DocumentSpan since our return type is language-agnostic.
+                documentSpans.Add(new DocumentSpan(document, location.SourceSpan));
```

Comment:
move this into the `else if (location.IsInSource)` section.  this shouldn't be done for metadata symbols or symbols with no locations.